### ISSUE TITLE
USB power sending garbage to Disting EX via TRS output

### ISSUE THREAD

**Issue by user9319062** [TomWhitwell/8mu_Public#9]
8mu only sends useful data down the MIDI TRS when plugged into my laptop. When I try a mains charger, or a power brick, it sends all sorts of garbage (stop/start messages, channel on/off messages etc) instead of the configured data. 

Video: https://drive.google.com/file/d/1BeTq3X8LhMQsimWfP_UUjrc40plyOlWr/view?usp=sharing) 
(Disting EX shows all the midi messages it receives.)

Tested the same TRS/wallplug combination with a Volca FM and all the faders worked properly. Also tried it with a crappy 'USB stick with DIN cables sticking out of it' MIDI interface, again powered by plug/mobile charger. All the CCs came in fine, per MidiView (mac). 

While the problem is limited to the Disting EX I'mm not sure whether it's an EX problem or an 8mu problem. Made a [post on modwiggler](https://www.modwiggler.com/forum/viewtopic.php?p=3969773#p3969773) to confirm with Expert Sleepers what might be causing the EX to receive data in this way, in case it helps establish what's going on

**Comment by user9319062** [comment]
From expert sleepers "I can't explain why the different power sources affect the behaviour, but I did have a look at the 8mu schematic, and its MIDI port doesn't comply with the MIDI electrical standard, which is surprising, and a possible source of error"

One thing i noted today is that it's not really 'garbage' showing up on the EX, it's a consistent interpolation of repeatable messages. The Disting only shows the last 4 items of MIDI history so I can't get a dump of what happens when you slide 1-127, but I manually transcribed a lot of them below, there's definitely some sort of pattern:

**1 - 70**
**Variations on the following, never 100% consistent, eg:**

1 - Ch 1 Off 12 0
2 - Ch 1 Off 3 48 
3 - Ch 1 Off 48 12
4 - Ch 1 Off 12 2
...etc 

**Then this pattern seems consitent:**

71 - Ch 1 Off 48 12
72 - Ch 2 Off 48 12
73 - Ch 3 Off 48 12
74 - Ch 4 Off 48 12
75 - Ch 5 Off 48 12
76 - Ch 6 Off 48 12
77 - Ch 7 Off 48 12
78 - Ch 8 Off 48 12
79 - Ch 9 Off 48 12
80 - Ch 10 Off 48 12
81 - Ch 11 Off 48 12
82 - Ch 12 Off 48 12
83 - Ch 13 Off 48 12
84 - Ch 14 Off 48 12
85 - Ch 15 Off 48 12
86 - Ch 16 Off 48 12
87 - Ch 1 On 48 12
88 - Ch 2 On 48 12
89 - Ch 3 On 48 12
90 - Ch 4 On 48 12
91 - Ch 5 On 48 12
92 - Ch 6 On 48 12
93 - Ch 7 On 48 12
94 - Ch 8 On 48 12
95 - Ch 9 On 48 12
96 - Ch 10 On 48 12
97 - Ch 11 On 48 12
98 - Ch 12 On 48 12
99 - Ch 13 On 48 12
100 - Ch 14 On 48 12
101 - Ch 15 On 48 12
102 - Ch 16 On 48 12
103 - Ch 1 Bend -6608
104 - Ch 2 Bend -6608
105 - Ch 3 Bend -6608
106 - Ch 4 Bend -6608
107 - Ch 5 Bend -6608
108 - Ch 6 Bend -6608
109 - Ch 7 Bend -6608
110 - Ch 8 Bend -6608
111 - Ch 9 Bend -6608
112 - Ch 10 Bend -6608
113 - Ch 11 Bend -6608
114 - Ch 12 Bend -6608
115 - Ch 13 Bend -6608
116 - Ch 14 Bend -6608
117 - Ch 15 Bend -6608
118 - Ch 16 Bend -6608
119 - SPP
120 - clock
121 - reserved
122 - start
123 - Continue
124 - Stop
125 - reserved
126 - Active Sensing
127 - Reset

In addition, I have a bank where all the accelerometer functions are disabled. When this bank is selected, no so-called 'garbage' is received from accelerometer movements. If I select a bank with CCs assigned to movements, they send out the pattern. So the 8mu is only sending data from fader/accelerometer movements that have been configured to do so (but the data is incorrect)

**Comment by expertsleepers** [comment]
I just tested with an 8mu, a disting EX, and the same PSU you mentioned on Modwiggler

https://www.amazon.co.uk/Huawei-Official-White-SuperCharge-Charger/dp/B07MCY64JD

and it all appears to work fine.

**Comment by user9319062** [comment]
Very mysterious. seems to suggest an issue with my particular unit?

I'm out of the country until next week but maybe I should try different wall power or something? Currently I think it's going into a 4-way and then the wall, but I'll have a mess around. If that doesn't work (I don't think it will lol) I can try to reset again, and then reinstall the firmware as well, see if that helps? 

I have to say, I was pretty sure I tried these steps initially. The thing is, the further away from the initial discovery I get, the less sure I am!

**Comment by user9319062** [comment]
OK, I've tried everything above, with no change in behaviour. Seems very peculiar that i appear to be the only one experiencing this. lucky me 😢 

While trying a few things out I did stumble on a workaround for when I want to perform live - the 8mu sends the correct CVs to the EX when connected to my phone (but not a power brick, sadly). This does mean I don't have to take my laptop with me if I want to leave the studio, which is a relief. Better than nothing!
